DEVELOPER.md

```diff
@@ -115,14 +117,22 @@ ipv6_prefix: "2001:bf7:860::/56"
 
 networks:
   - vid: 10                   # vlan-id
-    role: mesh                # what this vlan does (mesh vs. dhcp)
+    role: mesh                # what this vlan does (mesh, dhcp, mgmt)
     name: mesh_sama           # the name has a 12 characters limit. It should only contain lower letters and underscores
-    ptp: true                 # changing the mode from mesh to ether for reducing the airtraffic for point to point connections by ignoring the hidden node problem
+    ptp: true                 # changing the mode from mesh to ether for reducing the airtraffic for
+                              # point to point connections by ignoring the hidden node problem
     prefix: 10.31.42.16/32    # single ipv4-address for meshing
-    ipv6_subprefix: -10       # take an address from the back of the IPv6-block. Best practice is to use the same value as the vlan-id for everything except dhcp and mgmt to avoid duplicate addresses.
-    mesh_metric_lqm: ['default 0.8'] # link quality multiplier is used to artificially make routes worse for olsr, so certain links are preferred. Must be higher then 0.2, otherwise link wont work. Currently IPv4 routes over it.
-    mesh_metric: 1024         # overrides the default of 512 for for babel similar to the option above. Lower metrics means a route is preferred. Currently IPv6 routes over it.
-    untagged: true            # untags the vlan. It is commonly used for tunnel-uplinks
+    ipv6_subprefix: -10       # take an address from the back of the IPv6-block. Best practice is
+                              # to use the same value as the vlan-id for everything except dhcp and
+                              # mgmt to avoid duplicate addresses.
+    mesh_metric_lqm: ['default 0.8'] # link quality multiplier is used to artificially make routes
+                              # worse for olsr, so certain links are preferred. Must be higher then 0.2,
+                              # otherwise link wont work. Only used to connect to Falter-Routers.
+    mesh_metric: 1024         # overrides the default metrics for for babel routing.
+                              # Lower metrics means a route is preferred. Babel is used within bbb-configs.
+                              # Defaults can be found at group_vars/all/general.yml
+    untagged: true            # untags the vlan. It is commonly used for tunnel-uplinks. Only one
```

Comment:
Hier sollte man ggf. noch auf die Möglichkeit verweisen granulares untagging über rclocal commands vorzunehmen:
https://github.com/freifunk-berlin/bbb-configs/blob/main/locations/ilr.yml#L21-L27

Vielleicht sollte das Thema auch in einen eigenen Abschnitt.